src/app/kvm/components/KVMHeaderForms/AddLxd/SelectProjectForm/SelectProjectFormFields/SelectProjectFormFields.tsx

```diff
@@ -98,7 +100,7 @@ export const SelectProjectFormFields = ({
         />
         {projects.map((project) => {
```

Comment:
`project` implicitly has the `any` type

This is also true in SelectProjectForm.tsx:
https://github.com/petermakowski/maas-ui/blob/kvm-page-non-admin-error-4442/src/app/kvm/components/KVMHeaderForms/AddLxd/SelectProjectForm/SelectProjectForm.tsx#L62